contracts/BaseledgerUBTSplitter.sol

```diff
@@ -96,7 +96,7 @@ contract BaseledgerUBTSplitter is Context, Ownable {
         public
         emptyString(baseledgerDestinationAddress)
     {
-        require(amount > 0, "amount should be greater than zero");
+        require(amount >= 100000000, "amount should be >= 1");
```

Comment:
this way it is not configurable, we discussed owner function to change it potentially? or will always be 1?

Reply:
ah,  forgot about this. will expand.